Admin/FieldDescription.php

```diff
@@ -16,7 +16,7 @@
 class FieldDescription extends BaseFieldDescription
 {
     /**
-     * {@inheritdoc}
+     * Constructor.
```

Comment:
Which coding style ?

For symfony they don't add phpdoc where it doesn't had any meaningful information.